_layouts/default.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title> A6 Games | A6 Games</title>
+    {% feed_meta %}
+    {% seo %}
+</head> 
+<body>
+    <h1>A6 Games</h1>
+</body>
```

Comment:
So in general in template languages like HMTL / XML we indent inner tags in order to ensure we can quickly see which tags are contained in other ones. Also it looks like you may have already known that and this maybe was just a an oversight so sorry if I'm sounding patronizing 🤣. I know you said you were learning HTML so just wanted to clarify just in case.

So let's go ahead and indent everything inside the `html` tags 👍.